sshd_config

```diff
@@ -114,6 +120,11 @@ KexAlgorithms sntrup761x25519-sha512@openssh.com,curve25519-sha256,curve25519-sh
 
 ########## Connection Preferences ##########
 
+# Debian-based distributions only
+# hide the Debian banner to prevent information disclosure
+# (e.g., `SSH-2.0-OpenSSH_8.4p1 Debian-5+deb11u3`)
+#DebianBanner no
```

Comment:
It might be better to put this option in another config file, and introduce it from /etc/ssh/sshd_config.d/01-debianbanner.conf

Otherwise updating the main config would require users to resolve conflicts manually.

Reply:
While I agree with your point, it'll make downloading the config harder. Right now for new servers or even temporary servers I'll just do:

```bash
sudo curl -L k4t.io/sshd -o /etc/ssh/sshd_config
```

If there were to be an additional file more commands will have to be involved. I think we can leave it to the users to put this in `sshd_config.d` if they don't want it to be overwritten over an update.

Reply:
OK